demo/data/questions.json

```diff
@@ -84,6 +84,13 @@
         "collection"
       ],
       "queryFileName": "template11.sparql"
+    },
+    {
+      "text": "[TextBased] Get number of persons with a specific phenotype",
+      "variables": [
+        "plainText"
```

Comment:
I assume in a normal situation we would give the variable a more descriptive name?